doc-content/enterprise-only/installation/clustering-dg-proc.adoc

```diff
@@ -0,0 +1,40 @@
+[id='clustering-dg-proc_{context}']
+= Installing and configuring RHDG
+
+To use Red Hat JBoss Data Grid for more efficient searching across cluster nodes, you must install and configure Red Hat JBoss Data Grid for the {PRODUCT} clustered environment. Use the following instructions to configure a simplified, non-high availability environment.
+
+// To configure Red Hat JBoss Data Grid for a high availability {PRODUCT} clustered environment, see the https://access.redhat.com/documentation/en-us/red_hat_data_grid/7.2/html/administration_and_configuration_guide/high_availability_using_server_hinting[High Availability Using Server Hinting].
+// 
+// https://access.redhat.com/documentation/en-us/red_hat_data_grid/7.2/html/developer_guide/integration_with_eap[Integration with EAP]
+
+.Prerequisites
+* A Java Virtual Machine (JVM) environment compatible with Java 8.0 or later is installed. For more information about installing a JVM environment, see https://access.redhat.com/documentation/en-us/red_hat_data_grid/7.2/html/getting_started_guide/download_red_hat_jboss_data_grid#red_hat_jboss_data_grid_installation_prerequisites[Red Hat JBoss Data Grid Installation Prerequisites].
+* A backed-up {EAP} installation version 7.2 or higher is available. The base directory of the {EAP} installation is referred to as `__EAP_HOME__`.
+* {PRODUCT} is installed and configured.
+* Sufficient user permissions to complete the installation are granted.
+
+.Procedure
+. Navigate to the https://access.redhat.com/jbossnetwork/restricted/listSoftware.html[Software Downloads] page in the Red Hat Customer Portal (login required), and select the product and version from the drop-down options:
++
+* *Product:* Data Grid
+* *Version:* 7.2
+
+. Download and unzip the *Red Hat JBoss Data Grid 7.2.0 Server* (`jboss-datagrid-7.2.0-server.zip`) installation file to the preferred location on your machine.
++
+The unzipped directory is referred to as `$JDG_HOME`.
+. Patch Red Hat JBoss Data Grid 7.2.0 to 7.2.3, see https://access.redhat.com/documentation/en-us/red_hat_data_grid/7.2/html/7.2.3_release_notes/patching_red_hat_jboss_data_grid_server_instances[Patching Red Hat JBoss Data Grid Server Instances].
+. Run Red Hat JBoss Data Grid:
++
+* On Linux or UNIX-based systems, enter:
++
+[source]
+----
+$JDG_HOME/bin/standalone.sh
```

Comment:
here we need to add a parameter, so it will be:

`$JDG_HOME/bin/standalone.sh -c clustered.xml`

Reply:
`-c clustered.xml` parameter added.